### ISSUE TITLE
Generate KIE Output Json from Any LayoutLM Variant

### ISSUE THREAD

**Issue by purnasai-cyient** [NielsRogge/Transformers-Tutorials#182]
Hi @NielsRogge,

I am trying to convert the key, Value information that is generated from Any layoutLM variant into a JSON File. Need little help in doing that. Any Idea or way to convert to JSON.

Current output Format:
![image](https://user-images.githubusercontent.com/112476954/190893318-7e5fa8bb-cf85-435f-96bd-525d66e29934.png)

Required Format:
Map Question--->Answer:
{
"Fax":"614-466-5087",
"TO":"George Baroody",
"PHONE NUMBER":"(336) 335-7363",
"DATE":"12/10/98",
"NUMBER OF PAGES INCLUDING COVER SHEET:":"3"
}

If the above looks easy, I have a complex layout format like the below, then How do we do, key-value mapping better and efficiently?
![form-037-merchant-processing-application-tell-us-](https://user-images.githubusercontent.com/112476954/190893710-682e65e7-ac91-4906-9f34-cf3c46865cda.jpg)

![large](https://user-images.githubusercontent.com/112476954/190893716-859b3ce6-ebb3-4f8d-8346-4740c67158d3.png)

Thank you So much.

**Comment by jyotiyadav94** [comment]
Did you find an answer to get the KIE Output for the complex layout ?

**Comment by purnasai-cyient** [comment]
Not by following the above way of annotation. I changed the way I annotate. I need only a set of labels from the document. 

Currently, I annotated the required text for these classes and all other text as "others" labels. 

After Inference, I am doing post-processing to get those labels' text only.

**Comment by jyotiyadav94** [comment]
@purnasai-cyient 

Yeah I agree with few set of labels its easy to get extract the values.I haven't find a good approach for question & answer

**Comment by NielsRogge** [comment]
Hi,

I've uploaded some notebooks illustrating how to fine-tune LayoutLMv2/LayoutXLM for relation extraction here: https://github.com/NielsRogge/Transformers-Tutorials/tree/master/LayoutXLM
